src/main/resources/avro/variantmethods.avdl

```diff
@@ -64,16 +65,14 @@ org.ga4gh.models.VariantSet getVariantSet(
 /******************  /variants/search  *********************/
 /** This request maps to the body of `POST /variants/search` as JSON. */
 record SearchVariantsRequest {
-  /**
-  The VariantSet to search.
-  */
-  string variantSetId;
+  /** Required. The IDs of the variant sets to search over. */
+  array<string> variantSetIds = [];
 
```

Comment:
Same argument as variantSetIds above --- we only support one, so there's no point in reverting this.